webpack/assets/javascripts/react_app/components/Bookmarks/BookmarksReducer.js

```diff
@@ -3,14 +3,11 @@ import {
   BOOKMARKS_REQUEST,
   BOOKMARKS_SUCCESS,
   BOOKMARKS_FAILURE,
-  BOOKMARKS_MODAL_OPENED,
-  BOOKMARKS_MODAL_CLOSED,
   BOOKMARKS_FORM_SUBMITTED,
 } from './BookmarksConstants';
 import { STATUS } from '../../constants';
 
 export const initialState = Immutable({
-  showModal: false,
   currentQuery: '',
```

Comment:
I guess this is o longer needed as you get that query from a selector or from the parent component.

Reply:
@laviro I forgot another one!  Those things are everywhere 😆

Reply:
@laviro Updated, thanks